### ISSUE TITLE
Video being marked as played before completion

### ISSUE THREAD

**Issue by sarcinelli86** [sniklaus/youtube-watchmarker#86]
Hey guys, is there a way to set videos to be marked as watched only after finishing them?
The problem is that videos are marked as watched as soon as I play them.
![image](https://user-images.githubusercontent.com/34447967/178156364-1a100db4-01da-406e-b2c5-7a65c7230526.png)

**Here are my settings:**
![image](https://user-images.githubusercontent.com/34447967/178156386-ef30191f-195a-42cf-9a15-299d855ec7c6.png)

**Comment by sniklaus** [comment]
Does this video also show up in https://www.youtube.com/feed/history?

**Comment by sarcinelli86** [comment]
> Does this video also show up in https://www.youtube.com/feed/history?

Yes, it does.

**Comment by sniklaus** [comment]
Just to make sure I understand correctly: the video appears in your watch history on Youtube but it should not be part of the watch history in the watchmarker extension?

**Comment by sarcinelli86** [comment]
Yeah, that's right. Maybe it is new behavior on YouTube's side, the way they assign the Watched Badge now.
I remember videos being marked as watched only after you got past the midway point and not immediately as I hit play.
Before using the extension not necessarily all the videos that went to the Watch History were marked with the Watched Badge.

**Comment by sniklaus** [comment]
I am afraid that I am not sure how this can be implemented. The watchmarker extension syncs with your Youtube history in the background, so there would have to be a condition in the extension that says "only put videos in the extension's history if the Youtube history states that at least X% of the video has been watched". However, the Youtube history does not state how much of a video has been watched for older entries. I am hence under the impression that is impossible to fulfill this request. I am also under the impression that this is a duplicate of #65 which is why I am closing this issue for now.

**Comment by piecevcake** [comment]
If you want to watch a video you've opened later, you could leave it unloaded in another tab. That's what I do.
I would not want videos marked only after I finished watching, because it would not mark videos I started watching but don't want to watch.

> Hey guys, is there a way to set videos to be marked as watched only after finishing them?
